server/paymentProviders/transferwise/index.ts

```diff
@@ -122,20 +124,19 @@ async function quoteExpense(
   }
 
   const quote = await transferwise.createQuote(connectedAccount, quoteParams);
-
-  quote['paymentOption'] = quote.paymentOptions.find(p => p.payIn === 'BALANCE' && p.payOut === quote.payOut);
-  await expense.update({ data: { ...expense.data, quote: omit(quote, ['paymentOptions']) } });
-
-  return quote;
+  const paymentOption = quote.paymentOptions.find(p => p.payIn === 'BALANCE' && p.payOut === quote.payOut);
+  const expenseDataQuote = { ...omit(quote, ['paymentOptions']), paymentOption };
+  await expense.update({ data: { ...expense.data, quote: expenseDataQuote } });
+  return expenseDataQuote;
```

Comment:
@kewitz on this one I've added a new type, [`ExpenseDataQuoteV2`](https://github.com/opencollective/opencollective-api/pull/8340/files#diff-e1c593280c5ee11abbd0e23e6bfa7ecbe19714a6525ac50da9d2373b529da46dR72) that is simply removing `paymentOptions` in favor of `paymentOption`. I've therefore changed this snippet to make sure it always return such type.

So there's a real change here: before this edit, the `quoteExpense` function could return a quote with `paymentOptions` in case there was not quote yet for the expense (since the `omit` was only done in `expense.update`).

Let me know if this can cause an issue!

Reply:
I think this makes sense, nice work.